content/en/docs/tutorials/windows/windows-pods.md

```diff
@@ -3,15 +3,15 @@ reviewers:
 - jayunit100
 - jsturtevant
 - marosset
-title: Guide for scheduling Windows containers in Kubernetes
+title: Running Windows Workloads On A Kubernetes Cluster
 content_type: concept
-weight: 75
+weight: 20
 ---
 
 <!-- overview -->
 
 Windows applications constitute a large portion of the services and applications that run in many organizations.
-This guide walks you through the steps to configure and deploy Windows containers in Kubernetes.
+This tutorial walks you through the steps to configure and deploy Windows containers in Kubernetes.
```

Comment:
My understanding is that this page can be a task page rather than a tutorial. Just thinking loudly.

Reply:
What task is the reader performing?

Examples could be:
- on an existing cluster, run a standalone Pod that uses a Windows container image
- on an existing cluster, launch an application that uses a Windows container image
- set up a new cluster that includes Windows nodes

If the aim is for the reader to learn about Windows and Kubernetes more generally, that's a tutorial of sorts. Tasks should be about accomplishing a specific thing, possibly with defined variations on the main theme.

Reply:
If we agree that a task page is more appropriate, we may want to stop creating a tutorial for it at the moment.

Reply:
What do you recommend I do with https://kubernetes.io/docs/concepts/windows/user-guide/ in _this_ PR @tengqm?

Reply:
@sftim I'm more inclined to **have a task flavored page ,eg “Set up a Kubernetes cluster that includes Windows nodes”**.

Reply:
This PR is moving an existing page to tutorials section. What I was suggesting is to moving it to a task section. I don't think that leads to a huge PR. The PR grows large because it is trying to achieve several goals in it, which is not good already.

Reply:
I'll take the changes to this file out of the PR. The reduced scope may help us get it merged.